torch/distributed/distributed_c10d.py

```diff
@@ -513,6 +520,22 @@ def pg_coalesce_state(self) -> Dict[ProcessGroup, List[Union[_CollOp, P2POp]]]:
     def pg_default_device(self) -> Dict[ProcessGroup, torch.device]:
         return self._pg_default_device
 
+    @property
+    def pg_config_info(self) -> List[Dict[str, str]]:
+        config_info = []
+        for pg, backend in self.pg_map.items():
+            backend_type = Backend.backend_type_map[backend[0]]
```

Comment:
why backend[0]? Some clarification would be helpful if you think [0] is the right behavior.

Reply:
I'll add some comments. `backend` is a tuple with the first element being the backend type (nccl, etc.).

Reply:
Added comments.